core/src/main/java/edu/wpi/grip/core/metrics/Statistics.java

```diff
@@ -0,0 +1,92 @@
+package edu.wpi.grip.core.metrics;
+
+import com.google.common.base.MoreObjects;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Statistics analysis. Contains:
+ * <ul>
+ * <li>Number of samples ({@code n})</li>
+ * <li>Mean value ({@code mean})</li>
+ * <li>Standard deviation ({@code s})</li>
+ * </ul>
+ */
+public final class Statistics {
+
+  private final int n;
+  private final double sum;
+  private final double mean;
+  private final double s;
+
+  /**
+   * "null" statistics with every value set to zero.
+   */
+  public static final Statistics nil = new Statistics(0, 0, 0, 0);
+
+  /**
+   * Calculates the statistics of the given samples.
+   *
+   * @param samples the samples to analyze
+   * @return a statistical analysis of the given samples
+   */
+  public static Statistics of(double... samples) {
```

Comment:
I like this method! It keeps the constructor as it should be, simple assignments.